__tests__/app.test.js

```diff
@@ -68,4 +68,39 @@ describe('App', () => {
       });
     });
   });
+
+  describe('/api/articles/:article_id', () => {
+    test('GET:200 responds with a single article by article_id', () => {
+      return request(app)
+      .get('/api/articles/1')
+      .expect(200)
+      .then((response) => {
+        expect(response.body.article).toBeInstanceOf(Object)
+        expect(response.body.article.article_id).toBe(1);
+        expect(response.body.article.title).toBe('Living in the shadow of a great man');
+        expect(response.body.article.topic).toBe("mitch");
+        expect(response.body.article.author).toBe("butter_bridge");
+        expect(response.body.article.body).toBe("I find this existence challenging");
+        expect(response.body.article.created_at).toBe('2020-07-09T20:11:00.000Z');
+        expect(response.body.article.votes).toBe(100);
+        expect(response.body.article.article_img_url).toBe("https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700");
+      });
+    });
```

Comment:
A valid test, however, rather than multiple expects, we can have a single expect statement looking at the object itself e.g. `expect(response.body.article).toEqual({
 article_id: 1
 topic: "mitch"
 ... rest of object
})`

You don't have to change this btw. Just a suggestion. If you do change it, it isn't mega important and doesn't need to be done anytime soon 😄